sources/packages/api/src/route-controllers/esdc-integration/e-cert-integration.system.controller.ts

```diff
@@ -3,37 +3,67 @@ import { InjectLogger } from "../../common";
 import { LoggerService } from "../../logger/logger.service";
 import { AllowAuthorizedParty } from "../../auth/decorators";
 import { AuthorizedParties } from "../../auth/authorized-parties.enum";
-import { ECertFullTimeRequestService } from "../../esdc-integration/e-cert-full-time-integration/e-cert-full-time-request.service";
-import { ECertFullTimeResponseService } from "../../esdc-integration/e-cert-full-time-integration/e-cert-full-time-response.service";
-import { ESDCFileResultDTO, ESDCFileResponseDTO } from "./models/esdc-model";
+import { ECertFileHandler } from "../../esdc-integration/e-cert-integration/e-cert-file-handler";
+import { ESDCFileResponseDTO, ESDCFileResultDTO } from "./models/esdc-model";
 import { ApiTags } from "@nestjs/swagger";
 import BaseController from "../BaseController";
+import { OfferingIntensity } from "../../database/entities";
+import { ECertFullTimeResponseService } from "../../esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-full-time-response.service";
 
+const ECERT_FULL_TIME_FILE_CODE = "PBC.EDU.ECERTS.D";
```

Comment:
IMO these constants can be moved to _api\src\utilities\system-configurations-constants.ts_.